### PR TITLE
Fix a few things in the MTK benchmarks

### PR DESCRIPTION
**Pull request by ChrisRackauckas**

### CODE REVIEW

**Review by chriselrod — APPROVED** [review]

**Comment by ChrisRackauckas** [comment]
`rror: AssertionError: SciMLBase.successful_retcode(sol)` the implementation needs to be looked at more closely for correctness...

File: benchmarks/ModelingToolkit/RCCircuit.jmd [hunk 1]
```diff
@@ -187,6 +187,20 @@ end
 f
 ```
 
+```julia
+f2 = Figure(size = (800, 400));
+title = "Total Time: RC Circuit Benchmark"
+ax = Axis(f2[1, 1]; yscale = log10, xscale = log10, title)
+names = ["MTK", "JSIR - Scalar - Julia", "JSIR - Scalar - C", "JSIR - Scalar - LLVM", "JSIR - Loop - Julia", "JSIR - Loop - C", "JSIR - Loop - LLVM"], "OpenModelica", "Dymola"]
```
Comment (chriselrod) [inline benchmarks/ModelingToolkit/RCCircuit.jmd:194]:
```suggestion
names = ["MTK", "JSIR - Scalar - Julia", "JSIR - Scalar - C", "JSIR - Scalar - LLVM", "JSIR - Loop - Julia", "JSIR - Loop - C", "JSIR - Loop - LLVM", "OpenModelica", "Dymola"]
```

**Comment by chriselrod** [comment]
In RCCircuit, I saved the time and summed them for the total.
This PR instead calls `structural_simplify` again.

Want me to make a PR to change the approach?

**Comment by ChrisRackauckas** [comment]
Yes.

And let's look at the build at the end of the day, these two benchmarks should be good.

**Pull request merged**